src/Database/Driver/SqlDialectTrait.php

```diff
@@ -127,7 +127,7 @@ public function queryTranslator(string $type): Closure
      * objects to conform with the specific SQL dialect. Keys are class names
      * and values a method in this class.
      *
-     * @return array
+     * @return string[]
      */
```

Comment:
```suggestion
      @psalm-return array<class-string, string>
     */
```
I think we could get away with this here.

Reply:
I added it into it, just in case.